_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,462 @@
+array(8., dtype=float32)
+Coordinates:
+    spatial_ref  int64 0
+<xarray.DataArray ()>
+array(15497., dtype=float32)
+Coordinates:
+    spatial_ref  int64 0
+<xarray.DataArray ()>
+array(2477.405, dtype=float32)
+Coordinates:
+    spatial_ref  int64 0
+<xarray.DataArray ()>
+array(2061.9539, dtype=float32)
+Coordinates:
+    spatial_ref  int64 0
+~~~
+{: .output}
+
+And if we plot the image, the `nodata` pixels will show as white, since they are not 0 anymore:
+
+![Raster plot with rioxarray using the robust setting no data](../fig/E06-03-overview-plot-B09-robust-with-nan.png)
+
+## Raster Bands
+So far we looked into a single band, `B09` of a Sentinel-2 scene. However, to get a better understanding of the AoI, one may also want to visualize a true-color overview of the region. Then the single band will not be sufficient in this case.
+
+![Multi-band raster image](../images/dc-spatial-raster/single_multi_raster.png)
+
+The `overview` asset in the Sentinel-2 scene is a multiband asset. Similar to `B09`, we can load it by its `href`:
+~~~
+raster_ams_overview = rioxarray.open_rasterio(items[0].assets['overview'].href)
+~~~
+{: .language-python}
+~~~
+<xarray.DataArray (band: 3, y: 343, x: 343)>
+[352947 values with dtype=uint8]
+Coordinates:
+  * band         (band) int64 1 2 3
+  * x            (x) float64 6.002e+05 6.005e+05 ... 7.093e+05 7.096e+05
+  * y            (y) float64 5.9e+06 5.9e+06 5.899e+06 ... 5.791e+06 5.79e+06
+    spatial_ref  int64 0
+Attributes:
+    _FillValue:    0.0
+    scale_factor:  1.0
+    add_offset:    0.0
+~~~
+{: .output}
+
+The band number comes first when GeoTiffs are read with the `.open_rasterio()` function. As we can see in the `xarray.DataArray` object, the shape is `(band: 3, y: 343, x: 343)` now. There are three bands in the `xarray.DataArray` now. One can also check the shape in the `.shape` attribute:
+~~~
+raster_ams_overview.shape
+~~~
+{: .language-python}
+~~~
+(3, 343, 343)
+~~~
+{: .output}
+
+It's always a good idea to examine the shape of the raster array you are working with and make sure it's what you expect. Many functions, especially ones that plot images, expect a raster array to have a particular shape.
+
+One can visualize the multi-band data with the `DataArray.plot.imshow()` function:
+~~~
+raster_ams_overview.plot.imshow()
+~~~
+{: .language-python}
+
+![Amsterdam true color overview](../fig/E06-04-overview-plot-true-color.png)
+
+> ## Exercise: set the plotting aspect ratio
+> The true-color image we visualized is a little bit stretched. Can you visualize it with the right aspect? You can use the [Documentation](https://xarray.pydata.org/en/stable/generated/xarray.DataArray.plot.imshow.html) of `DataArray.plot.imshow()` for help.
+>
+>> ## Answers
+>> Since we know the height/width ratio is 1:1 (telling from the shape `(3, 343, 343)`), we can set the size of the image and force its ratio to be 1. For example, we can force the size to be 5 inches, and set `aspect=1`.
```

Comment:
Maybe we should also add that both the `aspect` and `size` arguments should be provided and how they are linked? 

```suggestion
>> Since we know the height/width ratio is 1:1 (check the `rio.height` and `rio.width` attributes), we can set the aspect ratio to be 1. For example, we can choose the size to be 5 inches, and set `aspect=1`.
```